apiserver/src/main/java/com/okbut/api/checkout/domain/Checkout.java

```diff
@@ -0,0 +1,38 @@
+package com.okbut.api.checkout.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name="CHECK_OUT")
+public class Checkout {
+    @Id
+    @GeneratedValue
+    @Column(nullable = false)
+    private Long id;
+    @CreatedDate
+    @Column(name="CHECKOUT_DTM", nullable = false)
+    private LocalDateTime checkoutDtm;
+    @Column(name="RETURN_DTM", nullable = false)
+    private LocalDateTime returnDtm;
+    @Column(name="IS_RENEWED", nullable = false)
+    private boolean isRenewed;
+    @ManyToOne
+    @JoinColumn(name="USER_ID")
+    @Column(nullable = false)
+    private Long userId;
+    @ManyToOne
+    @JoinColumn(name="BOOK_ID")
+    @Column(nullable = false)
+    private Long bookId;
```

Comment:
빈 줄을 둬서 가독성을 올리는 건 어떨까요?

```suggestion
    @Id
    @GeneratedValue
    @Column(nullable = false)
    private Long id;

    @CreatedDate
    @Column(name="CHECKOUT_DTM", nullable = false)
    private LocalDateTime checkoutDtm;

    @Column(name="RETURN_DTM", nullable = false)
    private LocalDateTime returnDtm;

    @Column(name="IS_RENEWED", nullable = false)
    private boolean isRenewed;

    @ManyToOne
    @JoinColumn(name="USER_ID")
    @Column(nullable = false)
    private Long userId;

    @ManyToOne
    @JoinColumn(name="BOOK_ID")
    @Column(nullable = false)
    private Long bookId;
```

Reply:
ㅋㅋㅋ수정했습니다